ui/pages/confirmations/hooks/useTransactionFunctions.js

```diff
@@ -201,9 +201,14 @@ export const useTransactionFunctions = ({
 
   const updateTransactionUsingEstimate = useCallback(
     (gasFeeEstimateToUse) => {
+      if (!gasFeeEstimates) {
+        return;
+      }
+
       if (!gasFeeEstimates[gasFeeEstimateToUse]) {
```

Comment:
Could we achieve the same with optional chaining here?

Reply:
True, pushed [here](https://github.com/MetaMask/metamask-extension/pull/29312/commits/18f8b8e79c74f5a00b38a8fdac06ad64ac15d0b1).